Formula/watson.rb

```diff
@@ -42,6 +44,20 @@ class Watson < Formula
 
   def install
     virtualenv_install_with_resources
+
+    if build.with? "completion"
+      bash_completion.install "watson.completion" => "watson"
+      zsh_completion.install "watson.zsh-completion" => "_watson"
+    end
+  end
+
+  def caveats
```

Comment:
Should by fine to nix the caveat here, since we're trying to decrease our usage of them.